.circleci/config.yml

```diff
@@ -149,30 +172,39 @@ jobs:
       - attach_workspace:
           at: /tmp/output
       - when:
-          condition: << parameters.production >>
+          condition:
+            equal: [ "integreat", << parameters.deliver_to >> ]
           steps:
             - run:
                 name: Read version number
                 command: echo 'export VERSION_NAME=$(cat version.json | jq .versionName)' >> $BASH_ENV
             - run:
-                name: Production delivery # StrictHostKeyChecking=no is not a security problem. The worst that could happen is a delivery to the wrong domain.
+                name: Integreat production delivery # StrictHostKeyChecking=no is not a security problem. The worst that could happen is a delivery to the wrong domain.
                 command: sftp -v -o StrictHostKeyChecking=no web@web.integreat-app.de:/integreat.app \<<< $'put -r /tmp/output/dist/*'
             - slack/notify:
                 message: Integreat WebApp version ${VERSION_NAME} was delivered successfully to integreat.app!
                 webhook: $SLACK_URL
-      - unless:
-          condition: << parameters.production >>
+      - when:
+          condition:
+            equal: [ "integreat-test-cms", << parameters.deliver_to >> ]
           steps:
             - run:
-                name: Debug delivery
+                name: Integreat debug delivery
                 command: sftp -v -o StrictHostKeyChecking=no web@web.integreat-app.de:/webnext.integreat-app.de \<<< $'put -r /tmp/output/dist/*'
+      - when:
+          condition:
+            equal: [ "malte", << parameters.deliver_to >> ]
+          steps:
+            - run:
+                name: Malte delivery (Nothing here yet) # Will be done in WEBAPP-558
+                command: echo "Skip delivery"
 
   bump_version:
     parameters:
-      shouldBumpVersion:
-        description:
-        type: boolean
-        default: true
+      build_config:
+        description: Which build config to use. Only bump version if build_config is "integreat".
```

Comment:
then why are "integreat-test-cms" and "malte" still part of the enum?

Reply:
We can't conditionally run jobs in circleci, therefore we have to run `bump_version` all the time we use a workflow which sometimes bumps a version and sometimes not, which is the case for the api_triggered workflow. Therefore, the passed build_config may also be malte and test cms.

Reply:
But we can conditonally run workflows... :thinking:  I guess we should do that then. Thanks!

Reply:
@Taggotty what do you think of my change?